src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -185,9 +189,10 @@ private void deleteRoles(Long userId) {
     private User getFromResultSet(ResultSet resultSet) throws SQLException {
         long id = resultSet.getLong("user_id");
         String name = resultSet.getString("user_name");
-        String login = resultSet.getString("login");
+        String login = resultSet.getString("user_login");
         String password = resultSet.getString("password");
-        User user = new User(name, login, password);
+        byte[] salt = resultSet.getBytes("user_salt");
+        User user = new User(name, login, password, getRoleSetByUserId(id), salt);
```

Comment:
we should not set roles here - we are opening one connection inside of another which is bad practice